### PR TITLE
Log args when process in janitor dies [skip tests]

### PR DESCRIPTION
**Pull request by palango**
Gives a bit more context

### CODE REVIEW

File: raiden/utils/nursery.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def subprocess_stopped(result: AsyncResult) -> None:
 
                         # if the subprocess error'ed propagate the error.
                         if result.get() != STATUS_CODE_FOR_SUCCESS:
-                            log.error("Process died! Bailing out.")
+                            log.error("Process died! Bailing out.", args=process.args)
```
Comment (karlb) [inline raiden/utils/nursery.py]: Shouldn't this be `process_args`?

Comment (palango) [reply raiden/utils/nursery.py]: Good point, why did this autocomplete?

Comment (palango) [reply raiden/utils/nursery.py]: Weird, do you know how `process` gets in scope in line 83?

Comment (karlb) [reply raiden/utils/nursery.py]: Yes, it's assigned in L94, which is in the same scope and executed first.

**Pull request merged**